packages/cli/cloud/src/deploy-project/action.ts

```diff
@@ -166,7 +155,38 @@ export default async (ctx: CLIContext) => {
     return;
   }
 
-  const cloudApiService = await cloudApiFactory(ctx);
+  const cloudApiService = await cloudApiFactory(ctx, token);
+
+  try {
+    const { data: projectInfo } = await cloudApiService.getProject({ name: project.name });
+    const isProjectSuspended = projectInfo.suspendedAt;
+
+    if (isProjectSuspended) {
+      ctx.logger.warn(
+        'Oops! This project has been suspended. \n Please reactivate it from the dashboard to continue deploying: '
+      );
+      ctx.logger.log(chalk.underline(`${apiConfig.dashboardBaseUrl}/projects/${project.name}`));
```

Comment:
Can we get this url from project info ?
As we discovered this morning, we might have changes in URLs in the future.

Reply:
Unfortunately, there is no dashboard url (or base url) in the project info. We would have to construct it in the same way. However, we could build it and return it on API side.

Reply:
Yes, that's what I meant 🙌

Reply:
I think I will do the same to fix the issue with the deployment link too.